Backend/Controllers/AuthenticationController.cs

```diff
@@ -79,4 +80,35 @@ public IQueryable<Session> GetSessions([FromAuthentication] User user)
     [HttpGet]
     public User GetAuthUser([FromAuthentication] User user) => user;
 
+    [HttpPost("reset/start")]
+    public async Task InitPasswordReset(string email) {
```

Comment:
Either use [FromBody] or use common Dto. FE uses inner class to match FromBody implementation, can easily be refactored to common model.
```suggestion
    public async Task InitPasswordReset([FromBody] string email) {```